common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java

```diff
@@ -233,6 +240,7 @@ public Builder queueCapacity(int queueCapacity) {
          * @param daemon whether the threads are daemon threads
          * @return updated builder instance
          */
+        @ConfiguredOption(value = "is-daemon", defaultValue = "true")
```

Comment:
`value` containing  config key name looks little confusing especially next to `defaultValue`

Reply:
Yes, I know. On the other hand this allows simple `@ConfiguredOption("key")`. I will check how often this would be used. If a lot, it makes sense to call it `value`, otherwise I will rename. it

Reply:
@ConfiguredOption("..") - 18 times
@ConfiguredOption(defaultValue="..") - 42 times
@ConfiguredOption(value = "", ...) - a lot

I will rename `value` to `key`.
We may also want to rename `defaultValue` to `value` as that is used much more often